src/compas_fab/backends/ros/backend_features/move_it_plan_motion.py

```diff
@@ -131,9 +137,9 @@ def plan_motion_async(self, callback, errback,
                        goal_constraints=goal_constraints,
                        path_constraints=path_constraints,
                        trajectory_constraints=trajectory_constraints,
-                       planner_id=options.get('planner_id', ''),
+                       planner_id=options.get('planner_id', 'RRTConnectkConfigDefault'),
                        group_name=group,
-                       num_planning_attempts=options.get('num_planning_attempts', 8),
+                       num_planning_attempts=options.get('num_planning_attempts', 1),
```

Comment:
This was a mismatch to the the documented default (but it worked because it was also defaulted at a higher layer).